main.c

```diff
@@ -216,7 +216,8 @@ bool default_filename(char *filename, size_t n, int filetype) {
 		ext = "jpeg";
 		break;
 #else
-		assert(false);
+		fprintf(stderr, "this version of grim doesn't support JPEG\n");
```

Comment:
No need for pretty error messages, we already have a user-friendly message on line 324.

Reply:
Oh, got it!

Reply:
Done